src/main/java/uk/gov/hmcts/cmc/claimstore/config/properties/notifications/EmailTemplates.java

```diff
@@ -138,4 +138,7 @@
 
     @NotEmpty
     private String claimantIntentionToProceedForOnlineDq;
+
+    @NotEmpty
+    private  String claimantSettledAfterFullDefense;
```

Comment:
Sorry, just noticed: should be `...Defence` ("defense" is the US spelling)

Reply:
Okay, I will try to watch out for that from now on.